rust/lance-index/src/vector.rs

```diff
@@ -19,6 +19,7 @@ pub mod residual;
 pub mod sq;
 pub mod transform;
 pub mod utils;
+pub mod v3;
```

Comment:
we skipped v2?

Reply:
we have v2 just not a package called `v2`